fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,318 @@
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/datum/martial_art/velvetfu/grab_act(mob/living/A, mob/living/D)
+	if(HAS_TRAIT(A, TRAIT_PACIFISM))
+		return FALSE
+	var/datum/dna/dna = A.has_dna()
+	if(dna?.check_mutation(HULK))
+		return FALSE
+	add_to_streak("G",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "grabbed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = pick("ascending claw", "descending claw")
+	var/bonus_damage = 10
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 5
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, A.get_attack_type())
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/datum/martial_art/velvetfu/harm_act(mob/living/A, mob/living/D)
+	var/datum/dna/dna = A.has_dna()
+	if(dna?.check_mutation(HULK))
+		return FALSE
+	add_to_streak("H",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "harmed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = "silken wrist"
+	var/bonus_damage = 10
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 5
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, A.get_attack_type())
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+
+/*
+ *	Help, Granters & Uplink Items
+ *
+ *	We are giving this item to Janitor traitors, since Alex is a trashman.
+ *	This is cheap, compared to other martial arts, but then again the martial art is a cheap knockoff.
+ *	The granter itself is a VHS tape, like the one Alex learns Velvet-Fu from.
+ */
+
+/mob/living/proc/velvetfu_help()
+	set name = "Recall Teachings"
+	set desc = "Remember the martial techniques of Velvet-Fu."
+	set category = "VelvetFu"
+
+	to_chat(usr, "<b><i>You try to remember the VHS tapes of Velvet-Fu...</i></b>\n\
+	<span class='notice'>Iron Hoof</span>: Disarm/Grab/Harm while opponent is down, though Disarm works best.\n\
+	<span class='notice'>Flying Axe Kick</span>: Harm Disarm. Deals damage and causes bleeding. Costs 50 Stamina.\n\
+	<span class='notice'>Goat Headbutt</span>: Disarm Grab. Deals brute while stunning your opponent, but hurts you. Costs 60 Stamina.\n\
+	<span class='notice'>Full Thrust</span>: Grab Harm. Deals brute and has a chance to knock your opponent down. Costs 60 Stamina.\n\
```

Comment:
you should probably mention that these all have a significant chance to fail

Reply:
and you should also probably mention that the normal disarm/grab/harm effects have been replaced by just damage